language-design.md

```diff
@@ -79,4 +101,6 @@ you do so? If not, why not?
 
 **Response**
 
+It does seem as if these two experiences are at odds with each other because Pavlus proposes that researchers found people preferred the natural-language syntax while Cook presents that in their experiment, this further confused programmers instead of resolving their issues with the language. I would only choose to include a small quantity of natural language when designing a DSL, such as for keywords, because this is common practice in programming languages and I would not propose a change to the structure programmers are used to unless I had evidence demonstrating that the programmers would prefer a change be made and that the decision I make would be more widely accepted, such as by user experiences and surverys. This follows in the work of Bloch who proposes that we should not make changes to APIs removing functionality users have grown accustomed to. I extend this to programming languages as a whole because it does seem like most programmers like to have a general set of functionality which is common across most general purpose programming languages.
```

Comment:
My take on this comparison was that the two experiences can be consistent, because in Quorum they were only tweaking the names of abstractions, while Cook describes more substantial adjustments to the syntax to make it more like human language. I think that the former works while the latter doesn’t because the human-inspired syntax makes computers out to be more flexible in interpreting statements than they really are. I definitely agree with you that using this human-inspired design for keywords (a la Quorum) is a good idea.